docs/source/serving/update.rst

```diff
@@ -1,42 +1,56 @@
 .. _serve-update:
 
-Update Your Service
-===========
+Updating a Service
+==================
+
+SkyServe supports *updating* a deployed service, which can be used to change:
+
+* Replica code (e.g., ``run``/``setup``; useful for debugging)
+* Replica resource spec (e.g., number of replicas; accelerator/instance used)
+* Service spec (e.g., autoscaling spec in ``replica_policy``)
 
-SkyServe supports update for your services. Use ``sky serve update`` to update an existing service:
+During an update, the service endpoint will remain the same and the service will
+remain accessible with no downtime.
+
+Use ``sky serve update`` to update an existing service:
 
 .. code-block:: console
 
     $ sky serve update service-name new_service.yaml
 
-SkyServe will launch new replicas described by ``new_service.yaml``. When the number of new replicas reaches the minimum number of replicas (``min_replicas``) required for the service, SkyServe will scale down old replicas to save cost. During the whole process, the service is still accessible to users. SkyServe allows users to update ``replica_policy`` parameters, such as ``target_qps_per_replica``. SkyServe also allows users to update ``resources`` parameters, such as ``cpu`` and ``memory``, so that new replicas can be launched on VMs of different types.   
+SkyServe will launch new replicas described by ``new_service.yaml`` with the following behavior:
+
+* An update is initiated, and traffic will still be redirected to existing (old) replicas.
+* New replicas (with new settings) are brought up in the background.
+* Once ``min_replicas`` new replicas are ready, traffic is *switched*: traffic will be sent to the new
```

Comment:
Thanks. Guess I didn’t understand the full picture. Is this correct: 

Let’s say enough new replicas just become ready; then new requests come in which will be redirected to new replicas. However, it’s possible that old replicas are still busy servicing some old requests. 

Does SkyServe wait for old requests to be completely finished (old replicas drained) before scaling down old replicas?

Reply:
For now the requests processing on the old replicas will be interrupted; we introduced node draining in #2793 in which it will wait 120 seconds before terminate the node.

Reply:
Reworded.